packages/venia-ui/lib/components/OrderHistoryPage/OrderDetails/shippingMethod.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { arrayOf, shape, string } from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import { mergeClasses } from '@magento/venia-ui/lib/classify';
+
+import defaultClasses from './shippingMethod.css';
+
+const ShippingMethod = props => {
+    const { data, classes: propsClasses } = props;
+    const { shipments, shippingMethod } = data;
+    const classes = mergeClasses(defaultClasses, propsClasses);
+    /**
+     * Shipments and Tracking are arrays. Since Venia does not
+     * support multiple shipping arrdresses in checkout we will
+     * be picking the first value in those arrays for now.
+     */
+    const [{ tracking }] = shipments;
+    const [{ carrier, number }] = tracking;
+
+    return (
+        <div className={classes.root}>
+            <div className={classes.heading}>
+                <FormattedMessage
+                    id="orderDetails.shippingMethodLabel"
+                    defaultMessage="Shipping Method"
+                />
+            </div>
+            <div className={classes.method}>{shippingMethod}</div>
+            <div className={classes.tracking}>
+                <FormattedMessage
+                    id="{carrier} Tracking: {number}"
```

Comment:
Use the dot notation id format please! 
```suggestion
                    id="shippingMethod.trackingInformation"
```